### PR TITLE
Updated facebook api to include a quote

### PR DESCRIPTION
**Pull request by tobyking3**
Updated the Facebook share api so that comments are shown. Switched from feed dialog to share dialog. Removed Deprecated Parameters.

### CODE REVIEW

File: src/models/Facebook.js [hunk 1]
```diff
@@ -23,10 +23,8 @@ define(['ShareToolsModel'], function (ShareToolsModel) {
             'redirect_uri': 'http://www.bbc.co.uk/news/special/shared/vj_sharetools/fb_red_uri.html?st_cb=facebook#state=feed',
             'display':      'popup',
             'locale':       'en_GB',
-            'link':         this.getShareUrl(),
-            'name':         this.getMessage().title,
-            'description':  this.getMessage().description,
-            'picture':      this.getMessage().image
```
Comment (ChrisBAshton) [inline src/models/Facebook.js:29]: Hi - do we not need to pass picture and description anymore? Does picture now come from the opengraph data in the parent page?

**Comment by ChrisBAshton** [comment]
Looks good to me 👍  one little question

File: src/models/Facebook.js (hunk 1 above)
Comment (tobyking3) [reply src/models/Facebook.js:29]: Picture and description are deprecated. I think the open graph tags do the work now.

Comment (ChrisBAshton) [reply src/models/Facebook.js:29]: Great - please merge, and bump the version number, `git tag` and also `npm publish`: https://www.npmjs.com/package/@bbc/news-vj-sharetools

Thanks!

**Pull request merged**